server/seed/seed.js

```diff
@@ -28,8 +27,8 @@ const writeUserRows = (writer, encoding, callback) => {
   write();
 }
 
-let continents = ['noam','noam','noam','eu','eu','soam','asia','asia','asia','aus','aus']
 const writeRentalRows = (writer, encoding, callback) => {
+  let continents = ['noam','noam','noam','eu','eu','soam','asia','asia','asia','aus','aus', 'ind', 'ind'];
```

Comment:
Just curious: why do you have multiple continents locations within the array? (ie: 3x noam, 2x eu, etc.)